scripts/ksql/run_ksql_commands.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+input=/tmp/ksqlcommands
+while IFS= read -r line
+do
+  /tmp/run_ksql_cmd.sh "$line"
```

Comment:
@rspurgeon why require `run_ksql_cmd.sh`?  Why not just put here the content of `run_ksql_cmd.sh`?

Reply:
> @rspurgeon why require `run_ksql_cmd.sh`? Why not just put here the content of `run_ksql_cmd.sh`?

IMO the small files are clearer to read, and the single function "run_ksql_cmd" is potentially useful independently.  I could do a function in the script also, but this works.  If you want it smashed up into a single line i'll change it.  LMK

Reply:
IMO it's better to mash it to avoid following a trail of files.   Thanks!

Reply:
Functional programmers might not agree ;-). I'll push a change